### ISSUE TITLE
[SUGGESTION] The ability to disable bonus loot drops and/or more preferably a way to replace the default loottable inside a/the config file.

### ISSUE THREAD

**Issue by FyzerVR** [HyperPigeon/Eldritch-Mobs#62]
Previously we did have a way to accomplish this with the use of "enableCustomLoot". If you did use this option you would then need to create a datapack with loot tables for each of the 3 category of boss type in the form of 3 json files. Once the datapack was properly assembled you then need to deploy it to each of the worlds you generate manually. 
While the old method did work it was cumbersome and expected a lot of specific knowledge of anyone of who wanted to be able disable or change the default loot table. It also adds complications/added steps to deployment between worlds/instances/packs.

Example: If say a player increase the difficulty by increasing the frequency of boss spawns by a large amount they might also want to turn down the quality and quantity of the default drops so as to not be flooded with enchanted diamond equipment and rare minerals.

TLDR:
An option to disable custom loot drops and/or a config option that provides customizable defaults.

Thank you for your time and hard work on this mod!

**Comment by HyperPigeon** [comment]
A config option has been added to disable bonus loot drops.
